HomebrewFormula/survex.rb

```diff
@@ -11,13 +12,32 @@ class Survex < Formula
   depends_on "gettext" => :build
   depends_on "pkg-config" => :build
 
+  head do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "netpbm" => :build
+    depends_on "w3m" => :build
+  end
+
   def install
+    if build.head?
+      system "cat /dev/null > doc/Makefile.am"
+      system "autoreconf", "-fiv"
+      system "git", "checkout", "INSTALL"
+      system "curl https://unifoundry.com/pub/unifont/unifont-14.0.01/font-builds/unifont-14.0.01.hex.gz | gzip -d > lib/unifont.hex"
+    end
+
     system "./configure", "--prefix=#{prefix}",
                           "--bindir=#{bin}",
                           "--mandir=#{man}",
                           "--docdir=#{doc}",
                           "--datadir=#{share}"
 
+    if build.head?
+      ENV.prepend_path "PATH", "/opt/homebrew/bin"
```

Comment:
In the CI we fail to find inkscape.  The log shows:

```
 ==> Downloading https://media.inkscape.org/dl/resources/file/Inkscape-1.1.2.dmg
Warning: macOS's Gatekeeper has been disabled for this Cask
==> Installing Cask inkscape
==> Moving App 'Inkscape.app' to '/Applications/Inkscape.app'
==> Linking Binary 'inkscape.wrapper.sh' to '/usr/local/bin/inkscape'
🍺  inkscape was successfully installed!
```

So should we actually be adding `/usr/local/bin` to `PATH` here instead?

Reply:
It seems this is a change in homebrew default for M1 macs vs x86 macs.  Trying to workaround...

Reply:
Yes, it is /opt/homebrew(bin on Apple Silicon (Arm) and /usr/local/bin on Intel. We can use `if Hardware::CPU.arm?` to run different commands on different CPU architectures. See https://rubydoc.brew.sh/Hardware/CPU.html

Reply:
It's simpler to just add both directories to `PATH`.

The inkscape part is working now, but I can't seem to get `Locale::PO` to be found:

https://github.com/ojwb/homebrew-survex/runs/5546356970?check_suite_focus=true

Reply:
`Installing /usr/local/Cellar/perl/5.34.0/lib/perl5/site_perl/5.34.0/POD2/PT_BR/local/lib.pod`

It looks like you have installed Perl via homebrew. I am using Apple shipped version 5.30.3. It is quite outdated but it works. I had a lot of problems with homebrew Perl installation and it is really not recommended. If you need up to date version on macOS use https://perlbrew.pl

Reply:
It's not what I've installed actually - it's installed by default in the github actions CI images for macos.

I've added `brew uninstall perl` so we're testing a more typical configuration, but that didn't fix it.

It seems homebrew runs formulae with `HOME` set to a temporary directory and no obvious way to know the real home directory, so I've changed things around to stop using `local::lib` and instead just install `Locale::PO` into that (temporary) home directory and point perl at it via `PERL5OPT`.

So `--HEAD` now works in CI, and should still for users (and the instructions are simpler as only `inkscape` needs to be installed by hand).